Lib/test/test_pathlib/test_pathlib_abc.py

```diff
@@ -1721,6 +1721,26 @@ def test_copy_symlink(self):
         self.assertFalse(target.is_symlink())
         self.assertEqual(source.read_text(), target.read_text())
 
+    @needs_symlinks
+    def test_copy_symlink_follow_symlinks_false(self):
+        base = self.cls(self.base)
+        source = base / 'linkA'
+        target = base / 'copyA'
+        source.copy(target, follow_symlinks=False)
+        self.assertTrue(target.exists())
+        self.assertTrue(target.is_symlink())
+        self.assertEqual(source.readlink(), target.readlink())
```

Comment:
Thoughts on using the `os` module functions directly to verify that `pathlib` is behaving properly, rather than using `pathlib` to verify itself?

```suggestion
        self.assertEqual(os.readlink(source), os.readlink(target)) # and the others...
```

Reply:
These tests run against a dummy subclass of `PathBase` in addition to `Path`, so use of `os` functionality is verboten. We do test that `Path.readlink()` calls `os.readlink()` correctly elsewhere, so I think it's OK.